app/presenters/publishing_api_presenters/case_study.rb

```diff
@@ -33,6 +33,14 @@ def details
     end
   end
 
+  def first_public_at
+    if edition.document.published?
+      edition.first_public_at
+    else
+      edition.document.created_at.iso8601
```

Comment:
Think this should be `updated_at`, so the editor has some indication that they are looking at their latest changes.